en/06_Project_Governance/07_Fixed_dependencies.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Fixed depnedencies
+---
+
+# Silverstripe CMS 5 fixed dependencies
+
+Silverstripe CMS relies on third party packages. Silverstripe CMS projects may interact directly or indirectly with those dependencies.
+
+We defined a list of "fixed dependencies" for the lifetime of Silverstripe CMS 5. Those dependencies will not be upgraded to new major version within the Silverstripe CMS 5 release line.
+
+Dependencies not in this list, may be changed or removed altogether within the lifetime of Silverstripe CMS 5.
+
+Dependency | Version
+-- | --
+composer-plugin-api | 2
```

Comment:
```suggestion
[composer-plugin-api](https://getcomposer.org/doc/articles/composer-platform-dependencies.md#plugin-package-composer-plugin-api) | 2
```
That's the closest I could find to a reference for what this pseudo package actually is